testsuite/MDAnalysisTests/utils/test_persistence.py

```diff
@@ -36,97 +38,136 @@
 from MDAnalysisTests.datafiles import PDB_small
 
 
-class TestAtomGroupPickle(TestCase):
-    def setUp(self):
-        """Set up hopefully unique universes."""
-        # _n marks named universes/atomgroups/pickled strings
-        self.universe = mda.Universe(PDB_small, PDB_small, PDB_small)
-        self.universe_n = mda.Universe(PDB_small, PDB_small, PDB_small,
-                                       anchor_name="test1")
-        self.ag = self.universe.atoms[:20]  # prototypical AtomGroup
-        self.ag_n = self.universe_n.atoms[:10]
-        self.pickle_str = cPickle.dumps(self.ag,
-                                        protocol=cPickle.HIGHEST_PROTOCOL)
-        self.pickle_str_n = cPickle.dumps(self.ag_n,
-                                          protocol=cPickle.HIGHEST_PROTOCOL)
-
-    def tearDown(self):
-        # Some tests might delete attributes.
-        try:
-            del self.universe
-            del self.universe_n
-            del self.ag
-            del self.ag_n
-        except AttributeError:
-            pass
-
-    def test_unpickle(self):
+class TestAtomGroupPickle(object):
+    """Set up hopefully unique universes."""
+
+    # _n marks named universes/atomgroups/pickled strings
+
+    @staticmethod
+    @pytest.fixture()
+    def universe():
+        return mda.Universe(PDB_small, PDB_small, PDB_small)
+    
+    @staticmethod
+    @pytest.fixture()
+    def universe_n():
+        return mda.Universe(PDB_small, PDB_small, PDB_small, anchor_name="test1")
+    
+    @staticmethod
+    @pytest.fixture()
+    def ag(universe):
+        return universe.atoms[:20]
+    
+    @staticmethod
+    @pytest.fixture()
+    def ag_n(universe_n):
+        return universe_n.atoms[:10]
+    
+    @staticmethod
+    @pytest.fixture()
+    def pickle_str(ag):
+        return cPickle.dumps(ag, protocol=cPickle.HIGHEST_PROTOCOL)
+    
+    @staticmethod
+    @pytest.fixture()
+    def pickle_str_n(ag_n):
+        return cPickle.dumps(ag_n, protocol=cPickle.HIGHEST_PROTOCOL)
+
+    
+
+    # def setUp(self):
```

Comment:
I haven't cleaned this up because the tests are still failing. Need to fix that first.